front/src/App/model/talk.js

```diff
@@ -0,0 +1,21 @@
+export default class Talk {
+  constructor(id, name, description, meetingLink, speaker) {
+    this.id = id;
+    this.name = name;
+    this.description = description;
+    this.meetingLink = meetingLink;
+    this.speaker = speaker;
+  }
+
+  checkIsAssigned(slots) {
+    this.isAssigned = slots.some((slot) => slot.talk.id === this.id);
+  }
+
+  checkIsInqueue(queue) {
+    this.isInqueue = queue.some((talk) => talk === this);
+  }
+
+  checkIsToSchedule(openSpace) {
+    this.isToSchedule = openSpace.toSchedule.some((talk) => talk.id === this.id);
+  }
```

Comment:
esto es 🎵 para mis 👂🏼 !!!
 
Lo único que te tiro es la siguiente pregunta: ¿se inicializa el objeto Talk en forma consistente?
Por ahí podrías recibir slots, queue, openSpace en el constructor.

E incluso podrías reutillizar la misma lambda: `(element) => element.talk === this` o `(element) => element.talk.id === this.id` para los 3 casos, cambiando únicamente la colección. Esto te lo tiro para un refactor futuro, no necesariamente ahora.